rs/bitcoin/kyt/src/main.rs

```diff
@@ -224,8 +345,10 @@ impl FetchEnv for KytCanisterEnv {
             }
             Err((r, m)) if is_response_too_large(&r, &m) => Err(HttpGetTxError::ResponseTooLarge),
             Err((r, m)) => {
-                // TODO(XC-158): maybe try other providers and also log the error.
-                println!("The http_request resulted into error. RejectionCode: {r:?}, Error: {m}");
+                log!(
+                    P1,
+                    "The http_request resulted into error. RejectionCode: {r:?}, Error: {m}"
```

Comment:
For this I think it would be useful to log the request as well so that we can try to reproduce it locally.

Reply:
Added `request` to the log